src/test/java/org/example/ServiceTests.java

```diff
@@ -0,0 +1,16 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+public class ServiceTests {
+
+    @Test
+    public void testAssertion() {
+        Service service = new Service();
+        assertThrows(AssertionError.class, () -> service.testAssertion());
```

Comment:
Can we assert on PR being created please?

Reply:
following method is in Service :  

public void testAssertion() {
        assert false : "Assertion failed";
    }

so in test it's created testAssertion.

Reply:
actually i created following class to failed assertion: Alpha